optuna/distributions.py

```diff
@@ -67,7 +83,11 @@ class UniformDistribution(
             Upper endpoint of the range of the distribution. ``high`` is excluded from the range.
     """
 
-    pass
+    def contains(self, param_value_in_internal_repr):
+        # type: (float) -> bool
+
+        value = param_value_in_internal_repr
+        return self.low <= value and value < self.high
```

Comment:
[TODO] This condition should be updated when https://github.com/pfnet/optuna/pull/384 is merged.